server/runtime/src/main/resources/schema/infinispan-server-15.2.xsd

```diff
@@ -404,6 +404,16 @@
                   <xs:documentation>Sets an external address for this node to accept client connections. Defaults to the server socket binding address.</xs:documentation>
                </xs:annotation>
             </xs:attribute>
+            <xs:attribute name="max-byte-array-size" type="xs:int">
```

Comment:
Add the source code default to the `attribute`.
Not sure if the name is correct, but something like this should work:
```suggestion
            <xs:attribute name="max-byte-array-size" type="xs:int" default="${ProtocolServer.max-byte-array-size}">
```